Code/src/main/java/nl/utwente/group10/haskell/hindley/HindleyMilner.java

```diff
@@ -25,6 +26,10 @@
      */
     static int tvOffset = 0;
 
+    static {
+        HindleyMilner.logger.setLevel(Level.OFF);
```

Comment:
I still don't like really like this, see #88.

You could change this to Level.WARNING and add a comment that changing it to Level.INFO will show debug messages, revert the change altogether, or configure a logging framework. I recommend option two.